lib/collection/request-body.js

```diff
@@ -70,6 +80,7 @@ _.assign(RequestBody.prototype, /** @lends RequestBody.prototype */ {
         // If mode is urlencoded but options does not provide any content, set it to an empty property list
         (mode === RequestBody.MODES.formdata && !formdata) && (formdata = new PropertyList(FormParam, this, []));
 
+        // @todo: Possibly use _.mergeDefined here?
```

Comment:
not yet. since it allows us to remove other modes' data for the time being. will not know the impact of this change. also, if we assume that this behaviour is expected, then the todo can be removed.